app/controllers/admin/routes.py

```diff
@@ -27,15 +28,18 @@ def trackVolunteersPage(programID, eventID):
     if g.current_user.isCeltsAdmin:
         if ProgramEvent.get_or_none(ProgramEvent.event == eventID, ProgramEvent.program == programID):
             eventParticipantsData = EventParticipant.select().where(EventParticipant.event == eventID)
+            eventRsvpData = EventRsvp.select().where(EventRsvp.event == eventID)
+            eventRsvpUsers = []
             eventParticipantsData = eventParticipantsData.objects()
```

Comment:
this line with .objects is unnecessary. (should be, unless I'm getting mixed up with peewee)

Reply:
you're right we don't need .object()